src/main/java/org/tools4j/hoverraft/state/TrackedFollowerState.java

```diff
@@ -45,4 +45,21 @@ public long nextIndex() {
     public long matchIndex() {
         return matchIndex;
     }
+
+    void nextIndex(final long index) {
+        this.nextIndex = index;
+    }
+
+    void decrementNextIndex() {
+        this.nextIndex--;
+    }
+
+    void resetMatchIndex() {
+        this.matchIndex = 0;
+    }
+    void matchIndex(final long index) {
```

Comment:
rename method as this is not expected to contain some logic, e.g.
``updateMatchIndexIfNeeded(..)``